pkg/api/server/v1alpha2/records_test.go

```diff
@@ -22,20 +22,22 @@ import (
 	"time"
 
 	"github.com/google/go-cmp/cmp"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
 	"github.com/tektoncd/results/pkg/api/server/db/pagination"
 	"github.com/tektoncd/results/pkg/api/server/test"
 	"github.com/tektoncd/results/pkg/api/server/v1alpha2/record"
 	recordutil "github.com/tektoncd/results/pkg/api/server/v1alpha2/record"
 	"github.com/tektoncd/results/pkg/api/server/v1alpha2/result"
 	resultutil "github.com/tektoncd/results/pkg/api/server/v1alpha2/result"
-	"github.com/tektoncd/results/pkg/internal/protoutil"
+	"github.com/tektoncd/results/pkg/internal/jsonutil"
```

Comment:
nit: given jsonutil only holds test helpers I am curious why we store it under `pkg/internal/` vs `test/`